gcloud/logging/config_service_v2_api.py

```diff
@@ -14,64 +14,123 @@
+                   'https://www.googleapis.com/auth/cloud-platform', )
+
+    _PROJECT_PATH_TEMPLATE = PathTemplate('projects/{project}')
+    _SINK_PATH_TEMPLATE = PathTemplate('projects/{project}/sinks/{sink}')
+
+    @classmethod
+    def project_path(cls, project):
+        """Returns a fully-qualified project resource name string."""
+        return cls._PROJECT_PATH_TEMPLATE.instantiate({'project': project, })
+
+    @classmethod
+    def sink_path(cls, project, sink):
+        """Returns a fully-qualified sink resource name string."""
+        return cls._SINK_PATH_TEMPLATE.instantiate({
+            'project': project,
+            'sink': sink,
+        })
+
+    def __init__(self,
+                 service_path=SERVICE_ADDRESS,
+                 port=DEFAULT_SERVICE_PORT,
+                 channel=None,
+                 ssl_creds=None,
+                 scopes=_ALL_SCOPES,
+                 retrying_override=None,
+                 bundling_override=None,
+                 timeout=30,
+                 app_name=None,
+                 app_version=None):
+        """Constructor.
+
+        Args:
+          :keyword service_path: The DNS of the API remote host.
+          :type service_path: string
+          :keyword port: The port on which to connect to the remote host.
+          :type port: int
+          :keyword channel: A Channel object through which to make calls.
+          :type channel: A grpc.beta.implementations.Channel object
+          :keyword ssl_creds: A ClientCredentials for use with an SSL-
+            enabled channel
+          :type ssl_creds: A grpc.beta.implementations.ClientCredentials
+            object
+          :keyword retrying_override: A dictionary that overrides default
+            retrying settings. ``retrying_override`` maps method names
+            (e.g., 'list_foo') to custom RetryOptions objects, or to None.
+            A value of None indicates that the method in question should not
+            retry.
+          :type retrying_override: dict
+          :keyword bundling_override: A dictionary that overrides default
+            bundling settings. ``bundling_override`` maps bundling method
+            names (e.g., 'publish_foo') to custom BundleOptions objects, or to
+            None. It is invalid to have a key for a method that is not
+            bundling-enabled. A value of None indicates that the method in
+            question should not bundle.
+          :type bundling_override: dict
+          :keyword timeout: The default timeout, in seconds, for calls made
+            through this client
+          :type timeout: int
+          :keyword app_name: The codename of the calling service.
+          :type app_name: string
+          :keyword app_version: The version of the calling service.
+          :type app_version: string
+        """
+        if app_name is None:
+            app_name = 'gax'
+        if app_version is None:
+            app_version = google.gax.__version__
```

Comment:
Keeping `__version__` as an importable constant is really a "religious preference" vs. keeping the constant in `setup.py`:  not work arguing over from a distance.